src/main/scala/io/getclump/package.scala

```diff
@@ -0,0 +1,15 @@
+package io
+
+package object getclump {
+
+  private[getclump] implicit val executionContext = scala.concurrent.ExecutionContext.global
```

Comment:
Ideally the execution context should be configurable. @stevenheidel do you think this approach is good enough for now?